docs/cloud-dev-env.md

```diff
@@ -0,0 +1,31 @@
+# Cloud Development Environments
+
+Altis has experimental built-in support for [GitHub Codespaces](https://github.com/features/codespaces), a Cloud-based development environment, that makes it possible to spin up a complete development environment in seconds within your browser, with a full editor based on Visual Studio Code and a fully-featured environment for your project, without having to install anything on your local machine.
+
+This is useful for many reasons:
+    - Bootstrap a full development environment in seconds.
+    - Zero local installation required.
+    - Standardized and highly configurable environment setup.
+    - Powerful and scalable development environment.
+    - Possibility of sharing preview environments with colleagues.
```

Comment:
Should we say “ability to share” rather than “possibility”? The latter sounds like it’s not completely straightforward in my mind when all that’s needed is to be logged into GH iirc